pex/pip.py

```diff
@@ -170,14 +195,21 @@ def maybe_trust_insecure_host(url):
             yield "--proxy"
             yield network_configuration.proxy
 
-        if network_configuration.cert:
-            yield "--cert"
-            yield network_configuration.cert
+        # N.B.: `--cert` is passed via env var to work around:
+        #   https://github.com/pypa/pip/issues/5502
+        # See `calculate_network_configuration_env`.
 
         if network_configuration.client_cert:
             yield "--client-cert"
             yield network_configuration.client_cert
 
+    def calculate_network_configuration_env(self, network_configuration=None):
```

Comment:
Shouldn't this method also be private? To keep consistent with the other `_calculate` that work as a "utility" method.

Reply:
Fixed.